readthedocs/doc_builder/python_environments.py

```diff
@@ -174,6 +175,7 @@ def is_obsolete(self):
 
         env_python = environment_conf.get('python', {})
         env_build = environment_conf.get('build', {})
+        envvars_hash = environment_conf.get('envvars_hash', None)
```

Comment:
nit: I prefer to call this variable/config `env_vars_hash`.

Reply:
Thanks.
I have pushed the changes.